sktime/forecasting/compose/_column_ensemble.py

```diff
@@ -147,7 +147,9 @@ def _predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
 
         y_pred = np.zeros((len(fh), len(self.forecasters_)))
         for (_, forecaster, index) in self.forecasters_:
-            y_pred[:, index] = forecaster.predict(fh)
+            y_pred[:, index] = forecaster.predict(
+                fh=fh, X=X, return_pred_int=return_pred_int, alpha=alpha
```

Comment:
`return_pred_int` and `alpha` are deprecated args, we should no longer be using these. I see how not passing `X` is a bug.

Reply:
regarding probabilistic prediction: `return_pred_int`, `alpha` has been replaced by `predict_interval`, and @kejsitake is currently working on precisely this, here:
https://github.com/alan-turing-institute/sktime/pull/2081

Reply:
FYI @kejsitake, there was n issue in `_predict` where `X` was not passed, which might be useful to be aware of